python/appsync-graphql-dynamodb/app_sync_cdk/app_sync_cdk_stack.py

```diff
@@ -0,0 +1,179 @@
+from aws_cdk import core
+from aws_cdk.aws_appsync import (
+    CfnGraphQLSchema,
+    CfnGraphQLApi,
+    CfnApiKey,
+    CfnDataSource,
+    CfnResolver
+)
+from aws_cdk.aws_dynamodb import (
+    Table,
+    Attribute,
+    AttributeType,
+    StreamViewType,
+    BillingMode
+)
+from aws_cdk.aws_iam import (
+    Role,
+    ServicePrincipal,
+    ManagedPolicy
+)
+
+class AppSyncCdkStack(core.Stack):
+
+    def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        table_name = 'items'
+
+        items_graphql_api = CfnGraphQLApi(
+            self, 'ItemsApi',
+            name='items-api',
+            authentication_type='API_KEY'
+        )
+
+        CfnApiKey(
+            self, 'ItemsApiKey', 
+            api_id=items_graphql_api.attr_api_id
+        )
+
+        api_schema = CfnGraphQLSchema(
+            self, 'ItemsSchema',
+            api_id=items_graphql_api.attr_api_id,
+            definition=f"""\
+                type {table_name} {{
+                    {table_name}Id: ID!
+                    name: String
+                }}
+                type Paginated{table_name} {{
+                    items: [{table_name}!]!
+                    nextToken: String
+                }}
+                type Query {{
+                    all(limit: Int, nextToken: String): Paginated{table_name}!
+                    getOne({table_name}Id: ID!): {table_name}
+                }}
+                type Mutation {{
+                    save(name: String!): {table_name}
+                    delete({table_name}Id: ID!): {table_name}
+                }}
+                type Schema {{
+                    query: Query
+                    mutation: Mutation
+                }}"""
+        )
+
+        items_table = Table(
+            self, 'ItemsTable',
+            table_name=table_name,
+            partition_key=Attribute(
+                name=f'{table_name}Id',
+                type=AttributeType.STRING
+            ),
+            billing_mode=BillingMode.PAY_PER_REQUEST,
+            stream=StreamViewType.NEW_IMAGE,
+
+            ## The default removal policy is RETAIN, which means that cdk destroy will not attempt to delete
+            ## the new table, and it will remain in your account until manually deleted. By setting the policy to 
+            ## DESTROY, cdk destroy will delete the table (even if it has data in it) 
+            removal_policy=core.RemovalPolicy.DESTROY ## NOT recommended for production code
```

Comment:
PEP8 - Inline comments should be preceded by two spaces, not one.  Also, use a single hash mark.